cicecore/cicedyn/infrastructure/ice_domain.F90

```diff
@@ -167,26 +170,41 @@ subroutine init_domain_blocks
    landblockelim     = .true.      ! on by default
 
    if (my_task == master_task) then
-      write(nu_diag,*) subname,' Reading domain_nml'
-
+      nml_name = 'domain_nml'
+      write(nu_diag,*) subname,' Reading ', trim(nml_name)
+      
       call get_fileunit(nu_nml)
       open (nu_nml, file=trim(nml_filename), status='old',iostat=nml_error)
       if (nml_error /= 0) then
          call abort_ice(subname//'ERROR: domain_nml open file '// &
-            trim(nml_filename), &
-            file=__FILE__, line=__LINE__)
+              trim(nml_filename), &
+              file=__FILE__, line=__LINE__)
```

Comment:
also, alignment across new abort_ice calls seems to vary.  could it be more consistent?